### ISSUE TITLE
Log4j Vulnerability

### ISSUE THREAD

**Issue by itsnotoger** [BTW-Community/BTW-Public#346]
It seems that old versions (prior to 1.18.1) of Minecraft were vulnerable to the log4j issue.
This meant that any player on a server could post a command in chat to execute any code on all player's systems.

Has this ever been patched in BTW? Is there even a need to do so? I'm not sure, but I couldn't find anything in the changelog, or issues about it.

I believe there are fixes to the client and server for this.

**Comment by SimonMeskens** [comment]
I thought Minecraft didn't use Log4j until later versions?

**Comment by Arminias** [comment]
Yes, pretty sure vanilla 1.5 does not use it. However, Fabric uses it and adds it as a dependency. We should probably have a look whether we need to update that dependency.

**Comment by LannyRipple** [comment]
Had to fix this at work.  BTW (Fabric) seems to be using log4j-core 2.8.1 and needs a version of 2.17.1+ to avoid the bug.  Latest log4j2 is 2.21.1.

**Comment by LannyRipple** [comment]
Ah.  I see it got set to 2.17.0 which, unfortunately still exhibits the bug.

**Comment by Arminias** [comment]
No, we use 2.19.0

**Comment by Arminias** [comment]
You need to get a new instance, we can not update instances.

**Comment by LannyRipple** [comment]
Doh!  Of course.  Thanks!

**Comment by Hirachosan** [comment]
I think we can close this @Arminias ?
